index.html

```diff
@@ -10,7 +10,14 @@
   </head>
   <body>
     <main>
-        <h1>Hello Microverse!-this is my first review</h1>
+      <h1>Hello Microverse!-this is my first review</h1>
+      <div class="div-p">this is a paragraph inside a div</div>
+      <ul class="ul-test">
+        <li>Item 1</li>
+        <li>Item 2</li>
+        <li>Item 3</li>
+        <li>Item 4</li>
+      </ul>
```

Comment:
Hello @NelsonAraujo93, 
It is a better HTML5 practice if you use a '<header>' tag and also containe the navbar elements in a '<nav>' tag.